### PR TITLE
[prow] Allow Case Insenstive Label Commands

### PR DESCRIPTION
**Pull request by grodrigues3**

### CODE REVIEW

File: prow/plugins/label/label_test.go [hunk 1]
```diff
@@ -120,6 +120,13 @@ func TestLabel(t *testing.T) {
 			commenter:         orgMember,
 		},
 		{
+			name:              "Adding Labels is Case Insensitive",
+			body:              "/kind BuG",
+			repoLabels:        []string{"area/infra", "priority/critical", "kind/bug"},
```
Comment (fejta) [inline prow/plugins/label/label_test.go]: Is it worth ensuring that `/kind bug` matches `repoLabels: "kind/BUG"`?

**Comment by crassirostris** [comment]
SGTM, thanks!

**Pull request merged**